t/ui/07-file.t

```diff
@@ -25,6 +25,7 @@ use lib "$FindBin::Bin/../lib";
 use Test::More;
 use Test::Mojo;
 use Test::Warnings;
+use File::Path qw(make_path remove_tree);
```

Comment:
Why not using Mojo::File ?

Reply:
File::Touch ist used for mtime in os-autoinst. If there's a better way to achieve the same I'm open for suggestions.

File::Path is already used elsewhere and hence present anyways. So why not using it here, too?

Reply:
For one simple reason: consistency See discussion in https://github.com/os-autoinst/os-autoinst/pull/1006#discussion_r207259933